libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/utils/QnACardBuilder.java

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.ai.qna.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.google.common.base.Strings;
+import com.microsoft.bot.ai.qna.models.QueryResult;
+import com.microsoft.bot.schema.ActionTypes;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.Attachment;
+import com.microsoft.bot.schema.CardAction;
+import com.microsoft.bot.schema.HeroCard;
+
+/**
+ * Message activity card builder for QnAMaker dialogs.
+ */
+public class QnACardBuilder {
+    /**
+     * Get active learning suggestions card.
+     *
+     * @param suggestionsList List of suggested questions.
+     * @param cardTitle       Title of the cards
+     * @param cardNoMatchText No match text.
+     * @return Activity.
+     */
+    public static Activity getSuggestionsCard(List<String> suggestionsList, String cardTitle, String cardNoMatchText) {
+        if (suggestionsList == null) {
+            throw new IllegalArgumentException("suggestionsList");
+        }
+
+        if (Strings.isNullOrEmpty(cardTitle)) {
```

Comment:
Here, in the [dotnet sample](https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs#L31), it verify only for null

Reply:
Done